test/integration/reachability.go

```diff
@@ -165,10 +165,35 @@ func (r *reachability) checkAccessLogs() error {
 			return nil
 		}
 
-		if n > budget {
-			return fmt.Errorf("exceeded budget for checking access logs")
+		time.Sleep(time.Second)
```

Comment:
yes, we need to wait between log fetching. otherwise, we just check 60 times in a second.